kano/utils/system.py

```diff
@@ -37,3 +39,47 @@ def is_systemd():
         return os.readlink('/sbin/init') == '/lib/systemd/systemd'
     except Exception:
         return False
+
+
+def get_kano_version_stamp():
+    '''
+    Returns the raw string from the version stamp file
+    '''
+    with open(kanux_stamp_file, 'r') as f:
+        return f.read().strip(' \n')
+
+
+def get_kano_version_date():
+    '''
+    Returns the date this version was built
+    '''
+    try:
+        return get_kano_version_stamp().split('-')[0].strip('Kanux')
```

Comment:
This works but it's pretty obscure since it relies on none of the names of days of the week starting with a letter in 'Kanux' :-)